.github/workflows/ngsa.yaml

```diff
@@ -111,7 +111,7 @@ jobs:
         docker run -d --name ngsa -p 4120:4120 test
         
         echo "Waiting for web server to start and run initial tests ..."
-        sleep 25
+        sleep 15
```

Comment:
Is it possible to use a readiness probe here on the container state instead of a sleep?

Reply:
Added to board